Task 1/Assets/Scripts/World/World.cs

```diff
@@ -0,0 +1,29 @@
+using StackApp.Blocks;
+using UnityEngine;
+
+namespace StackApp.World
+{
+    public class World : MonoBehaviour
```

Comment:
this class is really cohesive to GameController - and doesn't make much sense on it's own without GameController - i would suggest to remove it an move logic to GameController